# detection_mode: 
#   0 - normal speed read (0.03m ~ 5.6m)
#   1 - high speed read (0.01m ~ 1.1m) # recommended
detection_mode: 1
num_of_sensor: 1
serial_port: "/dev/ttyUSB0"
#serial_port: /dev/serial3by-id/usb-FTDI_FT232R_USB_UART_AB0LYXFN-if00-port0
serial_baud_rate: 115200

# default sonar index at each position
#     FRONT          
#    _2___1_         
# 3 |       | 8      
#   |       |        
# 4 |       | 7      
#    -------         
#   5      6         
#     BACK           
# Valid index (1~20) depends on the configuration
sonar_remapper: [1,2,3,4,5,6,7,8]